docs/web-apps/automated-testing/cypress.md

```diff
@@ -16,7 +16,6 @@ Supported OS:
 
 - Windows 10 / Windows 11
 - macOS 10.15+
-- Linux
```

Comment:
I think this list is meant to show what platforms can run `saucectl` and not what platforms on Sauce can run cypress tests. But this may not be relevant here at all. I don't mind just linking to https://docs.saucelabs.com/dev/cli/saucectl/ instead of duplicating the system reqs here.